desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
+        .query({ visible_to_public: 'true', term: value })
+        .set('X-ACCESS-TOKEN', sd.CURRENT_USER.accessToken)
+        .end((err, res) => {
+          if (!err || res.ok) {
+            dispatch(updateArtistSuggestions(res.body))
+            dispatch(hideNotConsigningMessage())
+          }
+        })
+    } catch (error) {
+      console.error('error!', error)
```

Comment:
Is there some standard thing we should do in case of failures like this? In general we don't want to show an error on the front-end (just _not_ return new values).